### PR TITLE
Rel 4.11.0.6

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+* [Revert "mb/pcengines/apu2: add reset logic for PCIe slots"](https://review.coreboot.org/c/coreboot/+/40147)
+
+**Total:**
+
+* 308 lines added,
+* 245 lines removed,
+
+in official coreboot repository.
+
+## Other news
```
Comment (miczyg1) [inline _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md:55]: @pietrushnic I came up with an idea to add another section which describes TrenchBoot related effort to the newsletter. Please have a look at this section and review, add suggestions, etc., if any.

File: _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+and trustworthy Next Generation Internet (NGI).
+
+To read more, please visit our blog describing the details and progress of our
+work which is based on the flagship PC Engines apu2 model:
+
+* [Introductory post](https://blog.3mdeb.com/2020/2020-03-28-trenchboot-nlnet-introduction/)
+* [Project basics](https://blog.3mdeb.com/2020/2020-03-31-trenchboot-nlnet-lz/)
+* [Validating the DRTM functionality](https://blog.3mdeb.com/2020/2020-04-03-trenchboot-nlnet-lz-validation/)
+
+If you are interested in the project follow us on our blog and [Twitter](https://twitter.com/3mdeb_com).
```
Comment (pietrushnic) [inline _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md:74]: @miczyg1 or contact us directly over email.

File: _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md (hunk 1 above)
Comment (pietrushnic) [reply _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md:55]: @miczyg1 very good idea IMO

File: _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md (hunk 2 above)
Comment (miczyg1) [reply _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md]: Done

File: _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md [hunk 3]
```diff
@@ -115,9 +115,9 @@ changes locally on our fork repository.
...
 
 * Mainline:
-  * PASSED: **433** (+0)
-  * FAILED: **22** (+0)
-  * PASSED [%]: **95.16** (+0%)
+  * PASSED: **437** (+4)
+  * FAILED: **14** (-8)
+  * PASSED [%]: **96.90** (+1.73%)
 
 No particular changes in tests in this release. Regression didn't detect new
```
Comment (miczyg1) [inline _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md:122]: @lwcislo a short commentary on the increase in total PASSED tests should be added

Comment (lwcislo) [reply _posts/2020-04-28-PC-Engines-Firmware-v4-11-0-6.md:122]: @miczyg1 done

**Pull request merged**